ui/pages/send/send-content/send-content.component.js

```diff
@@ -124,6 +140,36 @@ export default class SendContent extends Component {
     );
   }
 
+  renderRecipientWarning() {
+    const { acknowledgeRecipientWarning } = this.props;
+    const { t } = this.context;
+    return (
+      <div className="send__warning-container">
+        <ActionableMessage
+          type="danger"
+          useIcon
+          iconFillColor="#d73a49"
+          primaryActionV2={{
+            label: t('tryAnywayOption'),
+            onClick: () => acknowledgeRecipientWarning(),
```

Comment:
[REQ] just pass method name as it takes no props, the intermediate method is unnecessary.

Reply:
Fixed